python/fatcat_tools/importers/arabesque.py

```diff
@@ -165,14 +165,14 @@ def try_update(self, fe: FileEntity) -> bool:
 
         # TODO: this code path never gets hit because of the check above
         if set(fe.release_ids) == set(existing.release_ids):
-            existing_urls = set([u.url for u in existing.urls])
-            new_urls = set([u.url for u in fe.urls])
+            existing_urls = {u.url for u in existing.urls}
```

Comment:
I'm not a fan of the curly-bracket syntax in this context, replacing the explicit `set()` constructor. Is it possible to disable this rewrite with `pyupgrade`?

Reply:
pyupgrade is quite opinionated and does not include ways to switch off this conversion.

Eliminating function call overhead means that the proposed form is almost twice as fast.
```
from timeit import timeit

setup = "from string import ascii_letters"
fast = timeit("{x for x in ascii_letters}", setup=setup)
slow = timeit("set(x for x in ascii_letters)", setup=setup)
print(f"{fast} / {slow} = {fast / slow}")
print(f"{slow} / {fast} = {slow / fast}")
```
```
1.036221249960363 / 1.828327834024094 = 0.5667589973072124
1.828327834024094 / 1.036221249960363 = 1.7644183943284606
```
At first, I too liked the `set()` form but I have gotten used to reading the proposed form and I now prefer it.